packages/sdk-js/test/features.test.ts

```diff
@@ -1,4 +1,11 @@
 import { Context, GrowthBook } from "../src";
+/* eslint-disable */
```

Comment:
I'd 1) love feedback on how to clean this up, and 2) get confirmation that this won't have any adverse affects on bundle size or anything like that.